### PR TITLE
COMMONS-126: Close resource streams properly

### PR DESCRIPTION
**Pull request by thusithathilina**

### CODE REVIEW

**Comment by callkalpa** [comment]
Hi @thusithathilina,

Since this is for 4.5.x, can we look into the possibility of using streams in Java7?

File: components/deployment-synchronizer/org.wso2.carbon.deployment.synchronizer.subversion/src/main/java/org/wso2/carbon/deployment/synchronizer/subversion/util/SVNFileChecksumResolverUtil.java [hunk 1]
```diff
@@ -105,10 +105,7 @@ public static void resolveChecksum(String errorMessage) {
                 	}
                 	bw.newLine();                	
                 }
-                bis.close();                
-                fis.close();
-                bw.close();
-            }catch(Exception e){
+            } catch(Exception e){
                 log.error(e.getMessage());
```
Comment (Nipuni) [inline components/deployment-synchronizer/org.wso2.carbon.deployment.synchronizer.subversion/src/main/java/org/wso2/carbon/deployment/synchronizer/subversion/util/SVNFileChecksumResolverUtil.java]: formatting issue

File: components/logging/org.wso2.carbon.logging.view.ui/src/main/java/org/wso2/carbon/logging/view/ui/LogViewerClient.java [hunk 2]
```diff
@@ -90,6 +92,14 @@ public void downloadArchivedLogFiles(String logFile, HttpServletResponse respons
             String errorWhileDownloadingMsg = "Error while downloading file.";
             log.error(errorWhileDownloadingMsg, e);
             throw new LogViewerException(errorWhileDownloadingMsg, e);
+        } finally {
+            try {
+                if(fileToDownload != null) {
```
Comment (Nipuni) [inline components/logging/org.wso2.carbon.logging.view.ui/src/main/java/org/wso2/carbon/logging/view/ui/LogViewerClient.java]: formatting issue

File: components/deployment-synchronizer/org.wso2.carbon.deployment.synchronizer.subversion/src/main/java/org/wso2/carbon/deployment/synchronizer/subversion/util/SVNFileChecksumResolverUtil.java [hunk 3]
```diff
@@ -87,12 +88,11 @@ public static void resolveChecksum(String errorMessage) {
...
             log.debug("Trying to correct the checksum mismatch for SVN file:" +fullSvnFilePath+ "." +
             		"Expected:" +expectedChecksum.trim()+ " but it is:" +actualChecksum.trim());
-            try{
-                Reader fis = new FileReader(errorFile);
-                
-                BufferedReader bis = new BufferedReader(new InputStreamReader(new FileInputStream(errorFile)));
-                BufferedWriter bw = new BufferedWriter(new FileWriter(tmpFile));
-                
+
+            try (Reader fis = new FileReader(errorFile);
```
Comment (callkalpa) [inline components/deployment-synchronizer/org.wso2.carbon.deployment.synchronizer.subversion/src/main/java/org/wso2/carbon/deployment/synchronizer/subversion/util/SVNFileChecksumResolverUtil.java]: Is the ; at the end of the line correct?

**Pull request merged**